modules/specialized-guides/pages/large-deployments/hub-install.adoc

```diff
@@ -1,3 +1,5 @@
+// FIXME
+// 2024-10-15, ke: Do we still need this or parts of it?
```

Comment:
The installation is replaced by mgradm option --hubxmlrpc-replicas 1

The variables for /etc/hub/hub.conf can be customized via `Environment=` in systemd `/etc/systemd/system/uyuni-hub-xmlrpc.service.d/local.conf` (this is the same for all containers, I am not sure if it is documented)
HUB_API_URL is set automatically in the systemd unit file during installation.
HUB_CONNECT_USING_SSL is always enabled.